graphql-dgs/src/test/kotlin/com/netflix/graphql/dgs/DgsSchemaProviderTest.kt

```diff
@@ -890,6 +891,139 @@ internal class DgsSchemaProviderTest {
         }
     }
 
```

Comment:
Should we add a test that verifies validation does NOT happen when the schemaValidationWiring flag is set to false?